### PR TITLE
Improved recipe description handling

### PR DESCRIPTION
**Pull request by Firewolf1337**
- Recipe description will no longer show html markups.

- Added switches to hide the recipe description or limit the length of the text.
- Updated README and screenshot.

Hope you like it

Regard,
Thiemo

### CODE REVIEW

File: hacs.json [hunk 1]
```diff
@@ -1,5 +1,5 @@
 {
     "name": "Meal Plan Card - Lovelace style",
-    "homeassistant": "0.110.0",
+    "homeassistant": "0.111.0",
```
[the remark this answers is not in the document]
Comment (firstof9) [reply hacs.json:3]: We don't need to change this.

File: dist/grocy-meal-plan-card.js [hunk 2]
```diff
@@ -128,21 +129,20 @@ class MealPlanCard extends LitElement {
                       ? daily.type 
                       : ""}
                 </div>
-                <img class="pic" src="${daily.picture_url}"></img>
-            </div>
-            <div class=".info"> 
-                  ${daily.type === 'recipe' && typeof daily.recipe.description !== 'undefined' && daily.recipe.description !== null
-                  ?  (document.createElement('div').innerHTML = daily.recipe.description)
-                  : ""}
+                ${ daily.picture_url !== null ? `<img class="pic" src="${daily.picture_url}"></img>` : ""}
```
Comment (firstof9) [inline dist/grocy-meal-plan-card.js:132]: Mind throwing a `border-radius: 10px;` on the `pic` class? The rounded corners look nicer.

**Review by firstof9 — CHANGES_REQUESTED** [review]

**Comment by firstof9** [comment]
Thanks once more!

File: hacs.json (hunk 1 above)
[the remark this answers is not in the document]
Comment (Firewolf1337) [reply hacs.json:3]: ok thought that would be used as the version info.

File: dist/grocy-meal-plan-card.js (hunk 2 above)
Comment (Firewolf1337) [reply dist/grocy-meal-plan-card.js:132]: Yes that looks much better. I've set it to the same setting like the meal card.

**Review by firstof9 — APPROVED** [review]

**Comment by firstof9** [comment]
Thanks!

**Pull request merged**